cruise-control-core/src/main/java/com/linkedin/cruisecontrol/monitor/sampling/aggregator/MetricValues.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
```

Comment:
By default all the classes are not thread safe unless documented otherwise. So we don't need to say a class is not thread-safe. In our context, the MetricValues are part of the `MetricSampleAggregationResult`. So it is not going to be shared by multiple threads.